### ISSUE TITLE
Bugs in 2c_denoising and hdr_debevec

### ISSUE THREAD

**Issue by Brilbroeder** [florisdf/maibi_cv#2]
**hdr_debevec**

Replace
mask = np.zeros((height, width), dtype=np.bool8)
By
mask = np.zeros((height, width), dtype=np.bool_)

**2c_denoising_autoencoder**
for image_batch, _ in tqdm(train_loader):
ADD this line >>>>    image_batch = image_batch.to(device)

for image_batch, _ in tqdm(val_loader):
ADD this line >>>>  image_batch = image_batch.to(device)

I have a commit ready for a pull-request, but seems not allowed ;-)

**Comment by florisdf** [comment]
Thanks for opening this issue! Normally, pull requests should be allowed. It seems like you didn't fork the repository though. That's necessary to open a pull request. If you do so, I can merge your suggestions.

**Comment by florisdf** [comment]
See https://github.com/florisdf/maibi_cv/pull/3
